app/main.py

```diff
@@ -1 +1,22 @@
-# write your code here
+from typing import List
+from app.cafe import Cafe
+from app.errors import VaccineError, NotWearingMaskError
+
+
+
+def go_to_cafe(friends: List[dict], cafe: Cafe) -> str:
+    masks_to_buy = 0
+    for friend in friends:
+        try:
+            cafe.visit_cafe(friend)
+        except VaccineError:
+            # If any friend is unvaccinated or has an outdated vaccine
+            return "All friends should be vaccinated"
```

Comment:
Returning immediately inside the loop when a `VaccineError` is caught will terminate the function and not check the remaining friends. Consider accumulating errors or using a flag to determine if any friend is unvaccinated or has an outdated vaccine, and handle the return statement after the loop.